relayer/options.go

```diff
@@ -0,0 +1,31 @@
+package relayer
+
+import (
+	"github.com/strangelove-ventures/ibctest/ibc"
+)
+
+type RelayerOption interface{}
```

Comment:
For this pattern, people often add a sentinel, unexported method to the interface, such as

```go
type RelayerOption interface {
  relayerOption()
}
```

And then you would add the `relayerOption()` method, as a no-op,  to `RelayerOptionDockerImage` and `RelayerOptionImagePull`.

The advantage of this approach is that the compiler can be more restrictive about what values to allow than a type that just matches `interface{}`.